### DISCUSSION TITLE
WebAssembly support

### DISCUSSION THREAD

**Discussion by nicolodavis** [async-graphql/async-graphql#888] [General]
Can this library be used on the client-side?

I'm imagining a scenario where one might want to point their GraphQL client at a local schema running completely on the browser.

**Another way of phrasing the question:**
Is it possible to create an [Executable Schema](https://www.graphql-tools.com/docs/generate-schema) in Rust using async-graphql that runs on the browser?

**Comment by sunli829** [comment]
I think it's totally doable, basically you just need an async runtime that works inside wasm.
